service/storage/operations.go

```diff
@@ -190,7 +190,7 @@ func (l *Library) RetrievePayload(height uint64, path ledger.Path, payload *ledg
 		key := EncodeKey(PrefixPayload, path, height)
 		it := tx.NewIterator(badger.IteratorOptions{
 			PrefetchSize:   0,
-			PrefetchValues: true,
+			PrefetchValues: false,
```

Comment:
does this not work with prefetch?

Reply:
er eant prefetch to be a separate commit, so we know which one makes a bigger effect, before trying them together